processor/src/services/converters/helper.converter.ts

```diff
@@ -5,14 +5,15 @@ import {
   LineItem as CoCoLineItem,
   CustomLineItem,
   Address as CartAddress,
-  ShippingInfo,
   Order,
 } from '@commercetools/connect-payments-sdk';
 import {
   getAllowedPaymentMethodsFromContext,
   getCtSessionIdFromContext,
   getProcessorUrlFromContext,
 } from '../../libs/fastify/context/context';
+import { NormalizedShipping } from '@commercetools/connect-payments-sdk/dist/commercetools/types/cart.type';
```

Comment:
shouldn't this be imported like the others above?

Reply:
It seems that is not automatically exported the type in the payment-sdk. That will require an update in the payment-sdk.